src/App/Command/Release/MakeCommand.php

```diff
@@ -223,7 +200,7 @@ private function getCurrentVersion(GitWorkingCopy $git): Version
     private function getVersionToRelease(Version $currentVersion): Version
     {
         if ($this->tag === null) {
-            $versionType = $this->choose('What release is it?', '%s is not a valid release type.', Version::TYPES);
+            $versionType = $this->getIO()->choice('What release is it?', Version::TYPES);
```

Comment:
Custom error message is lost?

Reply:
Yes. It seemed to me that the standard SymfonyStyle message looks fine. Doesn't it?

![20 12 07-19 37 18](https://user-images.githubusercontent.com/6429469/101378082-b1efe800-38c3-11eb-87dd-7f29df4b832a.jpg)